src/tests/Sort.test.js

```diff
@@ -0,0 +1,43 @@
+const { app } = require('../pages/Application'); 
+
+describe('Inventory Sorting', () => {
+    before(async () => {
+        await app.login.navigate();
+        await app.login.performLogin();
+    });
+
+    it('should sort items by name (Z to A)', async () => {
+        await app.inventory.navigate();
+
+        const inventoryItems = await app.inventory.inventoryItems;
+        if (!inventoryItems.length) {
+            throw new Error('No inventory items found on the page');
+        }
```

Comment:
for this we have assertions e.g.
```
expect(inventoryItems.length).toBeGreaterThan(0)
```
read more [here](https://webdriver.io/docs/api/expect-webdriverio#default-matchers)